x/tokenfactory/keeper/createdenom.go

```diff
@@ -11,9 +11,12 @@ import (
 
 // ConvertToBaseToken converts a fee amount in a whitelisted fee token to the base fee token amount
 func (k Keeper) CreateDenom(ctx sdk.Context, creatorAddr string, subdenom string) (newTokenDenom string, err error) {
-	err = k.chargeForCreateDenom(ctx, creatorAddr)
-	if err != nil {
-		return "", err
+	// Don't charge denom creation fee for those in whitelist
+	if !(k.IsCreatorInDenomFeeWhitelist(ctx, creatorAddr)) {
+		err = k.chargeForCreateDenom(ctx, creatorAddr)
```

Comment:
we don't have to address this in this PR, but what do you think we should do about the charge for create funds if we're modifying the community fund?

Reply:
Yea this is something I'll discuss with tony on. Not really sure if we eliminate the community fund